internal/display/roles.go

```diff
@@ -63,7 +67,11 @@ func (r *Renderer) UserRoleList(roles []*management.Role) {
 	r.Heading(fmt.Sprintf("%s (%d)", resource, len(roles)))
 
 	if len(roles) == 0 {
-		r.Output("No roles assigned to user.\n\n")
+		if r.Format == OutputFormatJSON {
+			r.JSONResult([]interface{}{})
+			return
+		}
+		fmt.Fprintf(r.MessageWriter, "No %s.\n\n", resource)
```

Comment:
Should we keep the original "No roles assigned to user."?

Reply:
Yes, this was a mistake.

Reply:
Excuse me, this was actually intentional. `r.Output("...")` prints to stdout which is not consistent with the other commands. So while not exactly related to the json flag, this is a fix so we can apply stronger assertions to the no output tests.